src/drivers/osd/atxxxx/atxxxx.cpp

```diff
@@ -200,52 +173,46 @@ OSDatxxxx::init_osd()
 int
 OSDatxxxx::readRegister(unsigned reg, uint8_t *data, unsigned count)
 {
-	uint8_t cmd[5]; // read up to 4 bytes
-	int ret;
+	uint8_t cmd[5] {}; // read up to 4 bytes
 
 	cmd[0] = DIR_READ(reg);
 
-	ret = transfer(&cmd[0], &cmd[0], count + 1);
+	int ret = transfer(&cmd[0], &cmd[0], count + 1);
 
-	if (OK != ret) {
+	if (ret != PX4_OK) {
 		DEVICE_LOG("spi::transfer returned %d", ret);
 		return ret;
 	}
 
 	memcpy(&data[0], &cmd[1], count);
 
 	return ret;
-
 }
 
-
 int
 OSDatxxxx::writeRegister(unsigned reg, uint8_t data)
 {
-	uint8_t cmd[2]; // write 1 byte
-	int ret;
+	uint8_t cmd[2] {}; // write 1 byte
 
 	cmd[0] = DIR_WRITE(reg);
 	cmd[1] = data;
 
-	ret = transfer(&cmd[0], nullptr, 2);
+	int ret = transfer(&cmd[0], nullptr, 2);
 
 	if (OK != ret) {
 		DEVICE_LOG("spi::transfer returned %d", ret);
 		return ret;
 	}
 
 	return ret;
-
 }
 
 int
 OSDatxxxx::add_character_to_screen(char c, uint8_t pos_x, uint8_t pos_y)
 {
-
 	uint16_t position = (OSD_CHARS_PER_ROW * pos_y) + pos_x;
-	uint8_t position_lsb;
-	int ret;
+	uint8_t position_lsb = 0;
+	int ret = PX4_ERROR;
```

Comment:
This change is unnecessary.

Reply:
I know, but I'd rather initialize everything as a general rule.